<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOneSubjectDetail</class>
 <widget class="QWidget" name="AddOneSubjectDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加新题目</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>153</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>153</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>302</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>答案内容:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择分类:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="categoryBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>66</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>106</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>题目描述:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="4">
    <widget class="QTextEdit" name="questionEdit"/>
   </item>
   <item row="3" column="1" rowspan="2" colspan="4">
    <widget class="QTextEdit" name="answerEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
